src/environments/array.js

```diff
@@ -786,7 +786,31 @@ defineEnvironment({
         const symNode = checkSymbolNodeType(args[0]);
         const colalign: AnyParseNode[] =
             symNode ? [args[0]] : assertNodeType(args[0], "ordgroup").body;
-        const cols = colalign.map(function(nde) {
+        let myColAlign = [];
+        if (colalign[0].text === "*") {
```

Comment:
This seems to only support `*` at the beginning of `colalign`. But `*` can be used anywhere, e.g. `\begin{array}{ll*{5}{lcr}rr*5{rrcl}}`. I think ideally you'd do a `map` to expand all instances of `*`.